### DISCUSSION TITLE
Showcase on the list of Elysia plugins

### DISCUSSION THREAD

**Discussion by chneau** [m1212e/prismabox#17] [Ideas]
Hello,

I love the project! I would suggest to add this plugin to the list of plugins on the Elysia page: https://elysiajs.com/plugins/overview.html
which can be done here https://github.com/elysiajs/documentation/edit/main/docs/plugins/overview.md

**Comment by m1212e** [comment]
I've opened a PR: https://github.com/elysiajs/documentation/pull/362
Thanks for the idea!
